connexion/apis/abstract.py

```diff
@@ -31,8 +29,7 @@ def __init__(cls, name, bases, attrs):
         cls._set_jsonifier()
 
 
-@six.add_metaclass(AbstractAPIMeta)
-class AbstractAPI(object):
+class AbstractAPI(object, metaclass=AbstractAPIMeta):
```

Comment:
```suggestion
class AbstractAPI(metaclass=AbstractAPIMeta):
```
In python 3 all classes are newstyle classes, so we don't need `(object)` anywhere.